test/adapters/net_http_persistent_test.rb

```diff
@@ -10,8 +10,10 @@ def setup
         if defined?(Net::HTTP::Persistent)
           # work around problems with mixed SSL certificates
           # https://github.com/drbrain/net-http-persistent/issues/45
-          http = Net::HTTP::Persistent.new('Faraday')
-          http.ssl_cleanup(4)
+          if Net::HTTP::Persistent::instance_methods.include?(:ssl_cleanup)
+            http = Net::HTTP::Persistent.new('Faraday')
+            http.ssl_cleanup(4)
+          end
```

Comment:
(Investigating.)

Is this the current cleanup? https://github.com/drbrain/net-http-persistent/commit/dad5d682a6299e1bb9ef4fed3e2e5837011490bb

I begin to think that this _is_ the current way of "cleaning up" after finishing a connection.

Here's the `ssl_cleanup` implementation's comment (in v2.9.4):

>   ##
>   # Finishes all connections on the given +thread+ that were created before
>   # the given +generation+ in the threads +generation_key+ list.
>   #
>   # See #shutdown for a bunch of scary warning about misusing this method.